submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/DynamicExtSubmissionSubmitter.kt

```diff
@@ -0,0 +1,46 @@
+package ac.uk.ebi.biostd.submission.domain.service
+
+import ac.uk.ebi.biostd.common.properties.TaskHostProperties
+import ac.uk.ebi.biostd.submission.domain.extended.ExtSubmissionSubmitter
+import ac.uk.ebi.biostd.submission.domain.extended.LocalExtSubmissionSubmitter
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import mu.KotlinLogging
+import java.io.File
+import java.util.UUID
+
+private val logger = KotlinLogging.logger {}
+
+class DynamicExtSubmissionSubmitter(
+    private val extSubmissionSubmitter: LocalExtSubmissionSubmitter,
```

Comment:
I'm not very on board with the idea of the dynamic version containing the local one. I think we should have a handler that decides which one to use based on the configuration

Reply:
the problem is that we do not support yet any phase execution in submitter, only persistence. I guess we may in the future but I do not see the reason to complicate the implementation more  at this point.